libraries/3rdparty/mbedtls_utils/mbedtls_error.h

```diff
@@ -1,5 +1,5 @@
 /*
- * FreeRTOS Error Code Stringification utilities for mbed TLS v2.16.0
+ * FreeRTOS mbed TLS V0.1.0
```

Comment:
Is this the correct versioning for mbedtls utils?

Reply:
Yes, this was deliberate